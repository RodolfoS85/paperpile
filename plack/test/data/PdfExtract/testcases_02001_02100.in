---
file: ../../../PDFREPOSITORY/002001.pdf
---
file: ../../../PDFREPOSITORY/002002.pdf
---
file: ../../../PDFREPOSITORY/002003.pdf
---
file: ../../../PDFREPOSITORY/002004.pdf
---
file: ../../../PDFREPOSITORY/002005.pdf
---
file: ../../../PDFREPOSITORY/002006.pdf
---
file: ../../../PDFREPOSITORY/002007.pdf
---
file: ../../../PDFREPOSITORY/002008.pdf
---
file: ../../../PDFREPOSITORY/002009.pdf
---
file: ../../../PDFREPOSITORY/002010.pdf
---
file: ../../../PDFREPOSITORY/002011.pdf
---
file: ../../../PDFREPOSITORY/002012.pdf
---
file: ../../../PDFREPOSITORY/002013.pdf
---
file: ../../../PDFREPOSITORY/002014.pdf
---
file: ../../../PDFREPOSITORY/002015.pdf
---
file: ../../../PDFREPOSITORY/002016.pdf
---
file: ../../../PDFREPOSITORY/002017.pdf
---
file: ../../../PDFREPOSITORY/002018.pdf
---
file: ../../../PDFREPOSITORY/002019.pdf
---
file: ../../../PDFREPOSITORY/002020.pdf
---
file: ../../../PDFREPOSITORY/002021.pdf
---
file: ../../../PDFREPOSITORY/002022.pdf
---
file: ../../../PDFREPOSITORY/002023.pdf
---
file: ../../../PDFREPOSITORY/002024.pdf
---
file: ../../../PDFREPOSITORY/002025.pdf
---
file: ../../../PDFREPOSITORY/002026.pdf
---
file: ../../../PDFREPOSITORY/002027.pdf
---
file: ../../../PDFREPOSITORY/002028.pdf
---
file: ../../../PDFREPOSITORY/002029.pdf
---
file: ../../../PDFREPOSITORY/002030.pdf
---
file: ../../../PDFREPOSITORY/002031.pdf
---
file: ../../../PDFREPOSITORY/002032.pdf
---
file: ../../../PDFREPOSITORY/002033.pdf
---
file: ../../../PDFREPOSITORY/002034.pdf
---
file: ../../../PDFREPOSITORY/002035.pdf
---
file: ../../../PDFREPOSITORY/002036.pdf
---
file: ../../../PDFREPOSITORY/002037.pdf
---
file: ../../../PDFREPOSITORY/002038.pdf
---
file: ../../../PDFREPOSITORY/002039.pdf
---
file: ../../../PDFREPOSITORY/002040.pdf
---
file: ../../../PDFREPOSITORY/002041.pdf
---
file: ../../../PDFREPOSITORY/002042.pdf
---
file: ../../../PDFREPOSITORY/002043.pdf
---
file: ../../../PDFREPOSITORY/002044.pdf
---
file: ../../../PDFREPOSITORY/002045.pdf
---
file: ../../../PDFREPOSITORY/002046.pdf
---
file: ../../../PDFREPOSITORY/002047.pdf
---
file: ../../../PDFREPOSITORY/002048.pdf
---
file: ../../../PDFREPOSITORY/002049.pdf
---
file: ../../../PDFREPOSITORY/002050.pdf
---
file: ../../../PDFREPOSITORY/002051.pdf
---
file: ../../../PDFREPOSITORY/002052.pdf
---
file: ../../../PDFREPOSITORY/002053.pdf
---
file: ../../../PDFREPOSITORY/002054.pdf
---
file: ../../../PDFREPOSITORY/002055.pdf
---
file: ../../../PDFREPOSITORY/002056.pdf
---
file: ../../../PDFREPOSITORY/002057.pdf
---
file: ../../../PDFREPOSITORY/002058.pdf
---
file: ../../../PDFREPOSITORY/002059.pdf
---
file: ../../../PDFREPOSITORY/002060.pdf
---
file: ../../../PDFREPOSITORY/002061.pdf
---
file: ../../../PDFREPOSITORY/002062.pdf
---
file: ../../../PDFREPOSITORY/002063.pdf
---
file: ../../../PDFREPOSITORY/002064.pdf
---
file: ../../../PDFREPOSITORY/002065.pdf
---
file: ../../../PDFREPOSITORY/002066.pdf
---
file: ../../../PDFREPOSITORY/002067.pdf
---
file: ../../../PDFREPOSITORY/002068.pdf
---
file: ../../../PDFREPOSITORY/002069.pdf
---
file: ../../../PDFREPOSITORY/002070.pdf
---
file: ../../../PDFREPOSITORY/002071.pdf
---
file: ../../../PDFREPOSITORY/002072.pdf
---
file: ../../../PDFREPOSITORY/002073.pdf
---
file: ../../../PDFREPOSITORY/002074.pdf
---
file: ../../../PDFREPOSITORY/002075.pdf
---
file: ../../../PDFREPOSITORY/002076.pdf
---
file: ../../../PDFREPOSITORY/002077.pdf
---
file: ../../../PDFREPOSITORY/002078.pdf
---
file: ../../../PDFREPOSITORY/002079.pdf
---
file: ../../../PDFREPOSITORY/002080.pdf
---
file: ../../../PDFREPOSITORY/002081.pdf
---
file: ../../../PDFREPOSITORY/002082.pdf
---
file: ../../../PDFREPOSITORY/002083.pdf
---
file: ../../../PDFREPOSITORY/002084.pdf
---
file: ../../../PDFREPOSITORY/002085.pdf
---
file: ../../../PDFREPOSITORY/002086.pdf
---
file: ../../../PDFREPOSITORY/002087.pdf
---
file: ../../../PDFREPOSITORY/002088.pdf
---
file: ../../../PDFREPOSITORY/002089.pdf
---
file: ../../../PDFREPOSITORY/002090.pdf
---
file: ../../../PDFREPOSITORY/002091.pdf
---
file: ../../../PDFREPOSITORY/002092.pdf
---
file: ../../../PDFREPOSITORY/002093.pdf
---
file: ../../../PDFREPOSITORY/002094.pdf
---
file: ../../../PDFREPOSITORY/002095.pdf
---
file: ../../../PDFREPOSITORY/002096.pdf
---
file: ../../../PDFREPOSITORY/002097.pdf
---
file: ../../../PDFREPOSITORY/002098.pdf
---
file: ../../../PDFREPOSITORY/002099.pdf
---
file: ../../../PDFREPOSITORY/002100.pdf
