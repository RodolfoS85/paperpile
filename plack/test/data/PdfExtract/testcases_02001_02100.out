---
authors: 'Shirley Liu, X. and Brutlag, Douglas L. and Liu, Jun S.'
doi: 10.1038/nbt717
title: 'An algorithm for finding protein-DNA binding sites with applications to chromatin- immunoprecipitation microarray experiments'
---
authors: 'Lopez de Silanes, Isabel and Galban, Stefanie and Martindale, Jennifer L. and Yang, Xiaoling and Mazan-Mamczarz, Krystyna and Indig, Fred E. and Falco, Geppino and Zhan, Ming and Gorospe, Myriam'
doi: 10.1128/MCB.25.21.9520-9531.2005
title: Identification and Functional Outcome of mRNAs Associated with RNA-Binding Protein TIA-1
---
authors: 'Lorkovic, Zdravko J. and Lehner, Reinhard and Forstner, Christina and Barta, Andrea'
doi: 10.1261/rna.2440305
title: Evolutionary conservation of minor U12-type spliceosome between plants and humans
---
authors: 'Ma, Buyong and Elkayam, Tal and Wolfson, Haim and Nussinov, Ruth'
doi: 10.1073/pnas.1030237100
title: 'Protein-protein interactions: Structurally conserved residues distinguish between binding sites and exposed protein surfaces'
---
authors: 'Mallory, Allison C and Vaucheret, Hervé'
doi: 10.1038/ng1791
title: Functions of microRNAs and related small RNAs in plants
---
authors: 'Manning, G. and Whyte, D. B. and Martinez, R. and Hunter, T. and Sudarsanam, S.'
title: The Protein Kinase Complement of the Human Genome
---
authors: 'Maquat, Lynne E.'
title: 'Nonsense- mediated mRNA decay'
---
authors: 'Martin, William and Koonin, Eugene V.'
doi: 10.1038/nature04531
title: Introns and the origin of nucleus-cytosol compartmentalization
---
authors: 'Matter, Nathalie and Herrlich, Peter and Konig, Harald'
title: Signal-dependent regulation of splicing via phosphorylation of Sam68
---
authors: 'Milo, R. and Shen-Orr, S. and Itzkovitz, S. and Kashtan, N. and Chklovskii, D. and Alon, U.'
doi: 10.1126/science.298.5594.824
title: 'Network Motifs: Simple Building Blocks of Complex Networks'
---
authors: 'Miyazawa, Sanzo and Jernigan, Robert L.'
title: Identifying sequence-structure pairs undetected by sequence alignments
---
authors: 'Moll, Isabella and Afonyushkin, Taras and Vytvytska, Oresta and Kaberdin, Vladimir R. and Blasi, Udo'
doi: 10.1261/rna.5850703
title: Coincident Hfq binding and RNase E cleavage sites on mRNA and small regulatory RNAs
---
authors: 'Morgan, Kevin and Conklin, Darrell and Pawson, Adam J. and Sellar, Robin and Ott, Thomas R. and Millar, Robert P.'
doi: 10.1210/en.2002-220622
title: A Transcriptionally Active Human Type II Gonadotropin-Releasing Hormone Receptor Gene Homolog Overlaps Two Genes in the Antisense Orientation on Chromosome 1q.12
---
authors: 'Mousseau, Normand and Derreumaux, P and Gilbert, G'
doi: 10.1088/1478-3975/2/4/S04
title: Navigation and analysis of the energy landscape of small proteins using the activation-relaxation technique
---
authors: 'Ng, SEE-KIONG and Tan, SOON-HENG'
title: DISCOVERING PROTEIN-PROTEIN INTERACTIONS
---
authors: 'Ofran, Yanay and Rost, Burkhard'
doi: 10.1016/S0022-28360201223-8
title: Analysing Six Types of Protein-Protein Interfaces
---
authors: 'Ohler, Uwe and Niemann, Heinrich and Liao, Guo-chun and Rubin, Gerald M.'
title: Joint modeling of DNA sequence and physical properties to improve eukaryotic promoter recognition
---
authors: '{The FANTOM Consortium} and {the RIKEN Genome Exploration Research Group Phase I} and Team, II'
title: 'Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs'
---
authors: 'Olson, Wilma K. and Gorin, Andrey A. and Lu, XIANG-JUN and Hock, Lynette M. and Zhurkin, Victor B.'
title: DNA sequence-dependent deformability deduced from protein-DNA crystal complexes
---
authors: 'Poolsap, Unyanee and Kato, Yuki and Akutsu, Tatsuya'
title: DYNAMIC PROGRAMMING ALGORITHMS FOR RNA STRUCTURE PREDICTION WITH BINDING SITES
---
authors: 'Qiu, Xing and Xiao, Yuanhui and Gordon, Alexander and Yakovlev, Andrei'
doi: 10.1186/1471-2105-7-50
title: Assessing stability of gene selection in microarray data analysis
---
authors: 'Rehmsmeier, Marc and Steffen, Peter and Hochsmann, Matthias and Giegerich, Robert'
doi: 10.1261/rna.5248604
title: Fast and effective prediction of microRNA/target duplexes
---
authors: 'Rehwinkel, Jan and Natalin, Pavel and Stark, Alexander and Brennecke, Julius and Cohen, Stephen M. and Izaurralde, Elisa'
doi: 10.1128/MCB.26.8.2965-2975.2006
title: Genome-Wide Analysis of mRNAs Regulated by Drosha and Argonaute Proteins in Drosophila melanogaster
---
authors: 'Ruan, Jianhua and Zhang, Weixiong'
doi: 10.1186/1471-2105-6-114
title: 'CAGER: classification analysis of gene expression regulation using multiple information sources'
---
authors: 'Ryter, Jodi M. and Schultz, Steve C.'
title: 'Molecular basis of double-stranded RNA-protein interactions: structure of a dsRNA-binding domain complexed with dsRNA'
---
authors: 'Sabeti, P. C. and Schaffner, S. F. and Fry, B. and Lohmueller, J. and Varilly, P. and Shamovsky, O. and Palma, A. and Mikkelsen, T. S. and Altshuler, D. and Lander, E. S.'
title: Positive Natural Selection in the Human Lineage
---
authors: 'Sakai, H.'
title: EXTENSIVE SEARCH FOR DISCRIMINATIVE FEATURES OF ALTERNATIVE SPLICING
---
authors: 'Salari, Raheleh and Backofen, Rolf and Sahinalp, S Cenk'
doi: 10.1186/1748-7188-5-5
title: Fast prediction of RNA-RNA interaction
---
authors: 'Sashital, Dipali G and Cornilescu, Gabriel and Butcher, Samuel E'
doi: 10.1038/nsmb863
title: U2-U6 RNA folding reveals a group II intron-like domain and a four-helix junction
---
authors: 'Schones, Dustin E and Smith, Andrew D and Zhang, Michael Q'
doi: 10.1186/1471-2105-8-19
title: Statistical significance of cis-regulatory modules
---
authors: 'Schwartz, Scott and Kent, W. James and Smit, Arian and Zhang, Zheng and Baertsch, Robert and Hardison, Ross C. and Haussler, David and Miller, Webb'
doi: 10.1101/gr.809403
title: Human-Mouse Alignments with BLASTZ
---
authors: 'Schwarze, Ulrike and Starman, Barbra J. and Byers, Peter H.'
title: 'Redefinition of Exon 7 in the COL1A1 Gene of Type I Collagen by an Intron 8 Splice-Donor-Site Mutation in a Form of Osteogenesis Imperfecta: Influence of Intron Splice Order on Outcome of Splice-Site Mutation'
---
authors: 'Seelig, Burckhard and Jäschke, Andres'
title: A small catalytic RNA motif with Diels-Alderase activity
---
authors: 'Sen, Taner Z. and Kloczkowski, Andrzej and Jernigan, Robert L. and Yan, Changhui and Honavar, Vasant and Ho, Kai-Ming and Wang, Cai-Zhuang and Ihm, Yungok and Cao, Haibo and Gu, Xun and Dobbs, Drena'
doi: 10.1186/1471-2105-5-205
title: Predicting binding sites of hydrolase-inhibitor complexes by combining several methods
---
authors: 'Sethupathy, Praveen and Corda, Benoit and Hatzigeorgiou, Artemis G.'
doi: 10.1261/rna.2239606
title: 'TarBase: A comprehensive database of experimentally supported animal microRNA targets'
---
authors: 'Shalgi, Reut and Lieber, Daniel and Oren, Moshe and Pilpel, Yitzhak'
doi: 10.1371/journal.pcbi.0030131
title: Global and Local Architecture of the Mammalian microRNA-Transcription Factor Regulatory Network
---
authors: 'Shi, Hua and Hoffman, Bryan E. and Lis, John T.'
title: A Specific RNA Hairpin Loop Structure Binds the RNA Recognition Motifs of the Drosophila SR Protein B52
---
authors: 'Silverman, Scott K.'
doi: 10.1261/rna.2200903
title: 'Rube Goldberg goes (ribo)nuclear? Molecular switches and sensors made from RNA'
---
authors: 'Smith, Graham R and Sternberg, Michael JE'
title: Prediction of protein-protein interactions by docking methods
---
authors: 'Srinivasan, Gayathri and James, Carey M. and Krzycki, Joseph A.'
title: 'Pyrrolysine Encoded by UAG in Archaea: Charging of a UAG-Decoding Specialized tRNA'
---
authors: 'Starmer, J. and Stomp, A. and Vouk, M. and Bitzer, D.'
doi: 10.1371/journal.pcbi.0020057
title: Predicting Shine-Dalgarno Sequence Locations Exposes Genome Annotation Errors
---
authors: 'Stormo, Gary D. and Fields, Dana S.'
title: 'Specificity, free energy and information content in protein-DNA interactions'
---
authors: 'Styczynski, Mark P and Jensen, Kyle L'
title: 'An Extension and Novel Solution to the (l,d )-Motif Challenge Problem'
---
authors: 'Takai, Daiya and Jones, Peter A.'
doi: 10.1073/pnas.052410099
title: Comprehensive analysis of CpG islands in human chromosomes 21 and 22
---
authors: 'Terribilini, Michael and Lee, JAE-HYUNG and Yan, Changhui and Jernigan, Robert L. and Honavar, Vasant and Dobbs, Drena'
doi: 10.1261/rna.2197306
title: Prediction of RNA binding sites in proteins from amino acid sequence
---
authors: 'Thieffry, Denis and Salgado, Heladia and Huerta, Araceli M. and ColladoĆVides, Julio'
title: Prediction of transcriptional regulatory sites in the complete genome sequence of Escherichia coli K-12
---
authors: 'Urban, Johannes H. and Vogel, Jörg'
doi: 10.1007/978-1-59745-558-9_22
title: A Green Fluorescent Protein (GFP)-Based Plasmid System to Study Post-Transcriptional Control of Gene Expression In Vivo
---
authors: 'Uzilov, Andrew V and Keegan, Joshua M and Mathews, David H'
doi: 10.1186/1471-2105-7-173
title: Detection of non-coding RNAs on the basis of predicted secondary structure formation free energy change
---
authors: 'Valverde, Claudio and Livny, Jonathan and Schlüter, Jan-Philip and Reinkensmeier, Jan and Becker, Anke and Parisi, Gustavo'
doi: 10.1186/1471-2164-9-416
title: Prediction of Sinorhizobium meliloti sRNA genes and experimental detection in strain 2011
---
authors: 'van Oeveren, Jan and Janssen, Antoine'
doi: 10.1007/978-1-60327-411-1_4
title: Mining SNPs from DNA Sequence Data; Computational Approaches to SNP Discovery and Analysis
---
authors: 'Vasudevan, Shobha and Tong, Yingchun and Steitz, Joan A.'
title: 'Switching from Repression to Activation: MicroRNAs Can Up-Regulate Translation'
---
authors: 'Vecerek, Branislav and Moll, Isabella and Blasi, Udo'
doi: 10.1261/rna.2360205
title: Translational autocontrol of the Escherichia coli hfq RNA chaperone gene
---
authors: 'von Mering, Christian and Krause, Roland and Snel, Berend and Cornell, Michael and Oliver, Stephen G. and Fields, Stanley and Bork, Peer'
doi: 10.1038/nature750
title: Comparative assessment of large-scale data sets of protein-protein interactions
---
authors: 'Wen, Wu and Weiss, Sherri L. and Sunde, Roger A.'
title: UGA Codon Position Affects the Efficiency of Selenocysteine Incorporation into Glutathione Peroxidase-1
---
authors: 'Zamore, Phillip D. and Haley, Benjamin'
doi: 10.1126/science.1111444
title: 'Ribo-gnome: The Big World of Small RNAs'
---
authors: 'Zheng, Christina L. and Fu, XIANG-DONG and Gribskov, Michael'
doi: 10.1261/rna.2660805
title: Characteristics and regulatory elements defining constitutive splicing and different modes of alternative splicing in human and mouse
---
authors: 'Molkentin, Jeffery D. and Olson, Eric N.'
title: 'Combinatorial control of muscle development by basic helix-loop- helix and MADS-box transcription factors'
---
authors: 'Yeger-Lotem, Esti and Sattath, Shmuel and Kashtan, Nadav and Itzkovitz, Shalev and Milo, Ron and Pinter, Ron Y. and Alon, Uri and Margalit, Hanah'
doi: 10.1073/pnas.0306752101
title: Network motifs in integrated cellular networks of transcription-regulation and protein-protein interaction
---
authors: 'Li, Hao and Tang, Chao and Wingreen, Ned S.'
doi: 10.1002/prot.10239
title: 'Designability of Protein Structures: A Lattice-Model Study using the Miyazawa-Jernigan Matrix'
---
authors: 'Akutsu, Tatsuya'
title: Approximation and Exact Algorithms for RNA Secondary Structure Prediction and Recognition of Stochastic Context-free Languages
---
authors: 'Albaum, Stefan P. and Neuweger, Heiko and Franzel, Benjamin and Lange, Sita and Mertens, Dominik and Trotschel, Christian and Wolters, Dirk and Kalinowski, Jorn and Nattkemper, Tim W. and Goesmann, Alexander'
title: 'Qupe - a Rich Internet Application to take a Step Forward in the Analysis of Mass Spectrometry-Based Quantitative Proteomics Experiments'
---
authors: 'Ali Ansari-Lari, M. and Oeltjen, John C. and Schwartz, Scott and Zhang, Zheng and Muzny, Donna M. and Lu, Jing and Gorrell, James H. and Chinault, A. Craig and Belmont, John W. and Miller, Webb and Gibbs, Richard A.'
title: Comparative Sequence Analysis of a Gene-Rich Cluster at Human Chromosome 12p13 and its Syntenic Region in Mouse Chromosome 6
---
authors: 'Arnvig, Kristine B. and Young, Douglas B.'
doi: 10.1111/j.1365-2958.2009.06777.x
title: Identification of small RNAs in Mycobacterium tuberculosis
---
authors: 'Beaudry, Amber A. and Joyce, Gerald F.'
title: Minimum Secondary Structure Requirements for Catalytic Activity of a Self-splicing Group I Intron?
---
authors: 'Benson, Gary'
title: Sequence Alignment with Tandem Duplication
---
authors: 'Berghoff, Bork A. and Glaeser, Jens and Sharma, Cynthia M. and Vogel, Jörg and Klug, Gabriele'
doi: 10.1111/j.1365-2958.2009.06949.x
title: Photooxidative stress-induced and abundant small RNAs in Rhodobacter sphaeroides
---
authors: 'Bessière, Christian'
title: Arc-Consistency and Arc-Consistency Again
---
authors: 'Boiani, Michele and Schöler, Hans R.'
doi: 10.1038/nrm1744
title: REGULATORY NETWORKS IN EMBRYODERIVED PLURIPOTENT STEM CELLS
---
authors: 'Carmel, Ido and Tal, Saar and Vig, Ida and Ast, Gil'
doi: 10.1261/rna.5196404
title: Comparative analysis detects dependencies among the 5 splice-site positions
---
authors: 'Chen, Shuyan and Anderson, Karin and Moore, Melissa J.'
title: Evidence for a linear search in bimolecular 3 splice site AG selection
---
authors: 'Pires-daSilva, André and Sommer, Ralf J.'
doi: 10.1038/nrg977
title: THE EVOLUTION OF SIGNALLING PATHWAYS IN ANIMAL DEVELOPMENT
---
authors: 'Ding, Wei-Qun and Kuntz, Susan M. and Miller, Laurence J.'
title: 'A Misspliced Form of the Cholecystokinin-B/Gastrin Receptor in Pancreatic Carcinoma: Role of Reduced Cellular U2AF35 and a Suboptimal 3 -Splicing Site Leading to Retention of the Fourth Intron'
---
authors: 'Demaine, Erik D. and Mozes, Shay and Rossman, Benjamin and Weimann, Oren'
title: 'An O(n, )-Time Algorithm for Tree Edit Distance'
---
authors: 'Famulok, Michael and Mayer, Gunter and Blind, Michael'
doi: 10.1021/ar960167q
title: Nucleic Acid AptamerssFrom Selection in Vitro to Applications in Vivo
---
authors: 'Fletcher, Julia E. and Copeland, Paul R. and Driscoll, Donna M. and Krol, Alain'
title: 'The selenocysteine incorporation machinery: Interactions between the SECIS RNA and the SECIS-binding protein SBP2'
---
authors: 'Govindarajan, Sridhar and Goldstein, Richard A.'
title: Why are some protein structures so common?
---
authors: 'Grover, Andrew and Houlden, Henry and Baker, Matt and Adamson, Jennifer and Lewis, Jada and Prihar, Guy and Pickering-Brown, Stuart and Duff, Karen and Hutton, Mike'
title: 5 Splice Site Mutations in tau Associated with the Inherited Dementia FTDP-17 Affect a Stem-Loop Structure That Regulates Alternative Splicing of Exon 10
---
authors: 'Gutin, A. and Sali, A. and Abkevich, V. and Karplus, M. and Shakhnovich, E. I.'
title: "Temperature dependence of the folding rate in a simple protein model: Search for a ''glass'' transition"
---
authors: 'Hammell, Amy B. and Taylor, Ronald C. and Peltz, Stuart W. and Dinman, Jonathan D.'
title: Identification of Putative Programmed −1 Ribosomal Frameshift Signals in Large DNA Databases
---
authors: 'Hiller, Michael and Huse, Klaus and Platzer, Matthias and Backofen, Rolf'
doi: 10.1186/gb-2005-6-7-r58
title: 'Creation and disruption of protein features by alternative splicing - a novel mechanism to modulate function'
---
authors: 'Hofacker, Ivo L. and Stadler, Peter F.'
title: The Partition Function Variant of Sankoff 's Algorithm
---
authors: 'Jacquenet, Sandrine and Ropers, Delphine and Bilodeau, Patricia S. and Damier, Laurence and Mougin, Annie and Stoltzfus, C. Martin and Branlant, Christiane'
title: "Conserved stem-loop structures in the HIV-1 RNA region containing the A3 3' splice site and its cis-regulatory element: possible involvement in RNA splicing"
---
authors: 'Jensen, Iwan and Guttmann, Anthony J'
title: Statistics of lattice animals (polyominoes) and polygons
---
authors: 'Kent, W. James and Baertsch, Robert and Hinrichs, Angie and Miller, Webb and Haussler, David'
doi: 10.1073/pnas.1932072100
title: "Evolution's cauldron: Duplication, deletion, and rearrangement in the mouse and human genomes"
---
authors: 'Kranz, James K. and Hall, Kathleen B.'
title: RNA Binding Mediates the Local Cooperativity between the b-Sheet and the C-Terminal Tail of the Human U1A RBD1 Protein
---
authors: 'Kumar, Vipin'
title: 'Algorithms for Constraint- Satisfaction Problems: A Survey'
---
authors: 'Kurreck, Jens'
doi: 10.1155/JBB/2006/83757
title: 'siRNA Efficiency: Structure or Sequence - That Is the Question'
---
authors: 'Li, Binghui and Wachtel, Chaim and Miriami, Elana and Yahalom, Galit and Friedlander, Gilgi and Sharon, Gil and Sperling, Ruth and Sperling, Joseph'
doi: 10.1073/pnas.082095299
title: Stop codons affect 5 splice site selection by surveillance of splicing
---
authors: 'Liew, Chu Kong and Simpson, Raina J. Y. and Kwan, Ann H. Y. and Crofts, Linda A. and Loughlin, Fionna E. and Matthews, Jacqueline M. and Crossley, Merlin and Mackay, Joel P.'
doi: 10.1073/pnas.0407511102
title: 'Zinc fingers as protein recognition motifs: Structural basis for the GATA-1 Friend of GATA interaction'
---
authors: 'Lu, Kangmo and Lee, Mi-Hye and Yu, Hongwei and Zhou, Yuehua and Sandell, Shelley A. and Salen, Gerald and Patel, Shailendra B.'
title: 'Molecular cloning, genomic organization, genetic variations, and characterization of murine sterolin genes Abcg5 and Abcg8'
---
authors: 'Mallory, Allison C and Reinhart, Brenda J and Jones-Rhoades, Matthew W and Tang, Guiliang and Zamore, Phillip D and Barton, M Kathryn and Bartel, David P'
doi: 10.1038/sj.emboj.7600340
title: 'MicroRNA control of PHABULOSA in leaf development: importance of pairing to the microRNA 5, region'
---
authors: 'Maurer, Richard A. and Donelson, Christopher R. Erwin&and John E.'
title: Analysis of 5' Flanking Sequences and Intron-Exon Boundaries the Rat Prolactin Gene
---
authors: 'Moon, Kyung and Gottesman, Susan'
doi: 10.1111/j.1365-2958.2009.06944.x
title: A PhoQ/P-regulated small RNA regulates sensitivity of Escherichia coli to antimicrobial peptides
---
authors: 'Narberhaus, Franz and Vogel, Jörg'
doi: 10.1111/j.1365-2958.2009.06869.x
title: 'Regulatory RNAs in prokaryotes: here, there and everywhere'
---
authors: 'Newton, Derek C. and Bevan, Sian C. and Choi, Stephen and Brett Robb, G. and Millar, Adam and Wang, Yang and Marsden, Philip A.'
doi: 10.1074/jbc.M209988200
title: Translational Regulation of Human Neuronal Nitric-oxide Synthase by an Alternatively Spliced 5 -Untranslated Region Leader Exon
---
authors: 'Ott, S. and Tamada, Y. and Bannai, H. and Nakai, K. and Miyano, S.'
title: 'INTRASPLICING - ANALYSIS OF LONG INTRON SEQUENCES'
---
authors: 'Overgaard, Martin and Johansen, Jesper and Møller-Jensen, Jakob and Valentin-Hansen, Poul'
doi: 10.1111/j.1365-2958.2009.06807.x
title: Switching off small RNA regulation with trap-mRNA
---
authors: 'Overgaard, Martin and Kallipolitis, Birgitte and Valentin-Hansen, Poul'
doi: 10.1111/j.1365-2958.2009.06943.x
title: 'Modulating the bacterial surface with small RNAs: a new twist on PhoP/Q-mediated lipopolysaccharide modification'
---
authors: 'Peleg, Ofer and Brunak, SØREN and Trifonov, Edward N. and Nevo, Eviatar and Bolshoy, Alexander'
title: RNA Secondary Structure and Sequence Conservation in C1 Region of Human Immunodeficiency Virus Type 1 env Gene
---
authors: 'Pokala, Navin and Handel, Tracy M.'
doi: 10.1006/jsbi.2001.4349
title: "Review: Protein Design - Where We Were, Where We Are, Where We're Going"
